problems/11.Maximum Number of Pairs in Array.gs

```diff
@@ -1,25 +1,21 @@
 function numberOfPairs(nums: number[]): number[] {
-    let count = 0
-    let ostatak = 0
+
+    let pairCnt = 0
 
     for (let i = 0; i < nums.length - 1; i++) {
         for (let j = i + 1; j < nums.length; j++) {
             if (nums[i] == nums[j] && nums[i] != -1) {
-                count++
+                pairCnt++
                 nums[i] = -1
                 nums[j] = -1
 
             }
         }
     }
 
-    for (let i = 0; i < nums.length; i++) {
-        if (nums[i] != -1) {
-            ostatak++
-        }
-    }
+    let ostatak = nums.length - (pairCnt * 2)
```

Comment:
ostatak is unknown word, if you started using English and go with it (you have pairCnt)

One way to get around this you don't need to introduce new variable, you can just do:

return [pairCnt, nums.length - (pairCnt * 2)]

Also, this can be simplify to

return [pairCnt, nums.length - pairCnt * 2] 

Multiplication has higher order then subtraction - same as in math